Shared/src/shared/models/game/ServerTroop.java

```diff
@@ -0,0 +1,71 @@
+package shared.models.game;
+
+import shared.models.card.Card;
+import shared.models.game.map.Cell;
+
+public class ServerTroop extends Troop {
```

Comment:
Does this need to be in Shared or should it be in the server models instead? It could still `extends shared.models.game.Troop` from there.